Notes/Algorithms/Data Structures/Hash-Tables.c

```diff
@@ -337,9 +396,24 @@ void buildDeleteTests() {
     testDelete("KeyZero", 100, ERROR);
 }
 
-//main calls buildSearchTests, buildDeleteTests and buildInsertTests in succession to begin the process of testing the search, delete and insert functions
+int equalsEmpty(){
```

Comment:
Most of your other tests start with `test`.  Perhaps `testEqualsEmpty`?

Reply:
[commit e7c672394b8efb67073b7ebd78d75bcc0b815019](https://github.com/kcarrel/Training-Notes/pull/12/commits/e7c672394b8efb67073b7ebd78d75bcc0b815019)